### PR TITLE
RA-1809:Return of boolean expressions should not be wrapped into an i…

### PR DESCRIPTION
**Pull request by gracebish**
Description of what i changed
Return of boolean expressions should not be wrapped into an "if-then-else" statement"
Issue i worked on
https://issues.openmrs.org/browse/RA-1809

### CODE REVIEW

File: omod/src/main/java/org/openmrs/module/registrationapp/fragment/controller/summary/RegistrationSummaryFragmentController.java
Comment (HerbertYiga) [inline omod/src/main/java/org/openmrs/module/registrationapp/fragment/controller/summary/RegistrationSummaryFragmentController.java:94]: What about simply using (return extensionPointId.equals(extension.getExtensionPointId());

Comment (gracebish) [reply omod/src/main/java/org/openmrs/module/registrationapp/fragment/controller/summary/RegistrationSummaryFragmentController.java:94]: @HerbertYiga I thought storing a value in a variable first is easy to debug. Could you advise on the preference for openmrs so that i make changes?

Comment (HerbertYiga) [reply omod/src/main/java/org/openmrs/module/registrationapp/fragment/controller/summary/RegistrationSummaryFragmentController.java:94]: use return extensionPointId.equals(extension.getExtensionPointId());

Comment (gracebish) [reply omod/src/main/java/org/openmrs/module/registrationapp/fragment/controller/summary/RegistrationSummaryFragmentController.java]: @HerbertYiga Changes have been successfully made

**Review by HerbertYiga — APPROVED** [review]
Thanks grace cc @dkayiwa

**Comment by gracepotma** [comment]
@sacull can you approve this? Once that's done, @dkayiwa will merge :)

**Review by brandones — APPROVED** [review]

**Pull request merged**
